conf/robottelo.yaml.template

```
@@ -18,3 +18,6 @@ ROBOTTELO:
   SATELLITE_VERSION: "6.11"
   # The Base OS RHEL Version(x.y) where the satellite would be installed
   RHEL_VERSION: "7.9"
+  # Dynaconf and Dynaconf hooks related options
+  SETTINGS:
+    GET_FRESH: true
```

```suggestion
  REPOS:
    GET_FRESH: true
```
Also I would move this to `repos.yaml` as we getting fresh repos from ohsnap as I said above and not getting fresh robottelo settings anywhere from.